jib-core/src/main/java/com/google/cloud/tools/jib/image/LayerEntry.java

```diff
@@ -103,26 +100,6 @@ public FilePermissions getPermissions() {
     return permissions;
   }
 
-  // TODO: Remove these get...String methods.
-  /**
-   * Get the source file as an absolute path in Unix form. The path is made absolute first, if not
-   * already absolute.
-   *
-   * @return the source file path
-   */
-  public String getAbsoluteSourceFileString() {
-    return AbsoluteUnixPath.fromPath(sourceFile.toAbsolutePath()).toString();
-  }
-
-  /**
-   * Gets the extraction path as an absolute path in Unix form.
-   *
-   * @return the extraction path
-   */
-  public String getAbsoluteExtractionPathString() {
```

Comment:
Hmm, I dont think we should remove `getAbsoluteExtractionPathString` since it ends up just changing all calls to `getAbsoluteExtractionPathString()` into `getExtractionPath().toString()`.

Reply:
I don't really see that as a problem. I'd rather remove a method that wraps one method call to remove the extra level of indirection than avoid typing an extra dot.

Reply:
Yea, removing a one of the extraction path getters make sense. I feel like it makes more sense to remove the one that returns `AbsoluteUnixPath` but I guess `AbsoluteUnixPath` is straightforward enough to return.